src/commands/phpmyadmin.ts

```diff
@@ -124,6 +126,7 @@ export class PhpMyAdminCommand {
 	track: CommandTracker;
 	steps = {
 		ENABLE: 'enable',
+		PREPARING: 'preparing',
```

Comment:
Actually I've been thinking about this and I feel like maybe we should just keep this in the Enable step, no? I think preparing is ambiguous and isn't in line with the other steps

Reply:
I thought the same. My only discomfort is that it will be in the enable step for too long 😄

Reply:
I think that's fine

Reply:
@aswasif007 : I think you are correct. It is taking a _long_ time. Perhaps we should we should add some help text to the effect of "enabling PMA may take a long time in some situations"

Reply:
yeah that sounds good.

Reply:
"this step may take a while"

Reply:
https://github.com/Automattic/vip-cli/pull/1809/commits/87786ae03f0970a026500bc592eab20966bb0d70